apps/angular/projection/src/app/data-access/city.store.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, signal, WritableSignal } from '@angular/core';
+import { City } from '../model/city.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CityStore {
+  readonly cities: WritableSignal<City[]> = signal([]);
```

Comment:
you can create a readonly version of your signal if you want to protect if from the outside and set this one to private.

Reply:
i will keep that in mind! thanks